src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon(){
+        return (isDone ? "X" : " ");    //mark task with X or tick
+    }
+
+    public void markDone(){
+        this.isDone = true;     //mark task as done
```

Comment:
I like how there are comments given thoughout the program. I think to improve on this good practice one minor change can be made.

It might be better to leave [comments in their own lines](https://se-education.org/guides/conventions/java/basic.html#comments) with the same indentation level. 
Although the comments are short, giving them their own lines may be better practice to improve readability in the long run!

Reply:
Yes i totally agree with your comment. Left it there for debugging purposes and completely forgot about it!